javascript/builtins/Date.json

```
@@ -3052,14 +3052,7 @@
               "edge": "mirror",
               "firefox": {
                 "version_added": "preview",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "javascript.options.experimental.temporal",
-                    "value_to_set": "true"
-                  }
-                ],
-                "impl_url": "https://bugzil.la/1912511"
+                "impl_url": "https://bugzil.la/1946823"
```

I think we should point to the shipping bug now: https://bugzilla.mozilla.org/show_bug.cgi?id=1912511

```suggestion
                "impl_url": "https://bugzil.la/1912511"
```

---

Fixed in [ac5e61b](https://github.com/mdn/browser-compat-data/pull/26212/commits/ac5e61b5c8fcf336c5175591eed72d5ce89322f1), since @Elchi3 wrote the same.